CHANGELOG.md

```diff
@@ -2,10 +2,102 @@
+| [Country Risk](https://github.com/eclipse-tractusx/vas-country-risk-frontend)                | country-risk: [1.0.0](https://github.com/eclipse-tractusx/vas-country-risk-frontend/releases/tag/country-risk-1.0.0) | Frontend: [v1.1.2](https://github.com/eclipse-tractusx/vas-country-risk-frontend/releases/tag/v1.1.2)<br/> Backend: [v1.1.1](https://github.com/eclipse-tractusx/vas-country-risk-backend/releases/tag/v1.1.1) |
+| [Digital Twin KIT](https://eclipse-tractusx.github.io/docs-kits/category/digital-twin-kit)                                              |                                                           n/a                                                            |          [1.0.0](https://eclipse-tractusx.github.io/docs-kits/kits/Digital%20Twin%20Kit/Digital%20Twin%20Kit%20Changelog)           |
+| [Digital Twin Registry](https://github.com/eclipse-tractusx/sldt-digital-twin-registry)                                                |                     registry: [0.3.24](https://github.com/eclipse-tractusx/sldt-digital-twin-registry/releases/tag/registry-0.3.24)                     |                                                                                                                                                     [v0.3.16-M1](https://github.com/eclipse-tractusx/sldt-digital-twin-registry/releases/tag/v0.3.16-M1)                                                                                                                                                     |
+| [Eclipse Data Space Connector (EDC)](https://github.com/eclipse-tractusx/tractusx-edc)                                                 |                  tractusx-connector: [0.5.3](https://github.com/eclipse-tractusx/tractusx-edc/tree/refs/tags/tractusx-connector-0.5.1)                  |                                                                                                                                                                 [0.5.3](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.5.1)                                                                                                                                                                 |
+| [Golden Record Business Partner Number (BPN) Service](https://github.com/eclipse-tractusx/bpdm)                                        |                                     bpdm: [3.0.4](https://github.com/eclipse-tractusx/bpdm/releases/tag/bpdm-3.0.4)                                     |                                                                                                                                                                    [v4.0.1](https://github.com/eclipse-tractusx/bpdm/releases/tag/v4.0.1)                                                                                                                                                                    |
```

Comment:
guess theese are the corrected version information for golden record / bpdm

| NAME |   CHART VERSION |  APP VERSION | 
|-|-|-|
|tractusx-dev/bpdm   |    3.1.2   |        4.1.0      |

Helm Chart (s)
bpdm: [3.1.2](https://github.com/eclipse-tractusx/bpdm/releases/tag/bpdm-3.1.2)

App-/KIT Version (s)
[v4.1.0](https://github.com/eclipse-tractusx/bpdm/releases/tag/v4.1.0)